tracer/test/snapshots/AwsLambdaTests.verified.txt

```diff
@@ -0,0 +1,272 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: http.request,
+    Resource: GET localhost/function/HandlerNoParamAsync,
+    Service: Bootstrap-http-client,
+    Type: http,
+    ParentId: Id_3,
```

Comment:
Span with `Id_3`, which is the span for the Lambda function itself is missing from the traces. This is expected because it's created by the AWS _extension_ (outside of the .NET process), so it's sent independently of the Tracer. 

The error here is benign, the HTTP request is simply being used to create a child span to demonstrate that the lambda is actually being called, and that it has the correct properties. The endpoint it's calling doesn't exist, hence the error.